### PR TITLE
Add impl for WasefireWrite

### PR DESCRIPTION
**Pull request by kofls**

### CODE REVIEW

File: examples/rust/opensk/Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ persistent_store = { path = "../../../third_party/OpenSK/libraries/persistent_st
 rand_core = "0.6.4"
 wasefire = { path = "../../../crates/prelude" }
 wasefire-stub = { path = "../../../crates/stub", optional = true }
+wasefire-logger = { path = "../../../crates/logger" }
```
Comment (ia0) [inline examples/rust/opensk/Cargo.toml:14]: This crate is only for platform implementation. For applets, you have the `debug!()` macro.

File: examples/rust/opensk/src/write.rs [hunk 2]
```diff
@@ -12,11 +12,23 @@
...
-    fn write_str(&mut self, _: &str) -> core::fmt::Result {
-        todo!()
+    fn write_str(&mut self, data: &str) -> core::fmt::Result {
+        self.buf.write_str(data)
+    }
+}
+
+impl Drop for WasefireWrite {
+    fn drop(&mut self) {
+        log::debug!(self.buf)
```
Comment (ia0) [inline examples/rust/opensk/src/write.rs:32]: This should be `wasefire::debug!("{}", self.buf)`.

File: examples/rust/opensk/Cargo.toml (hunk 1 above)
Comment (kofls) [reply examples/rust/opensk/Cargo.toml]: Removed this.

File: examples/rust/opensk/src/write.rs (hunk 2 above)
Comment (kofls) [reply examples/rust/opensk/src/write.rs]: Done.

**Review by ia0 — APPROVED** [review]

**Comment by ia0** [comment]
cc @kaczmarczyck

**Pull request merged**